types/tests.ts

```diff
@@ -180,6 +180,9 @@ const runAdmin = async () => {
 
   await admin.listTopics()
 
+  await admin.fetchOffsets({ groupId: 'test-group', topic: 'topic1' })
```

Comment:
Maybe toss an `await admin.fetchOffsets({ groupId })` in there as well to show that `topics` is optional.